### ISSUE TITLE
Ministry User Logs In

### ISSUE THREAD

**Issue by michesmi** [bcgov/SIMS#659]
As a (Ministry User) I want (Log in) So That (I can Support Students and Institutions)

**Additional Context**
When viewing an application from the student application display an application details screen. This page will give the student all the information relevant to what is happening with their application
- This is intended to be the backend authentication pathway to unblock future work 

**Describe the task**
Create login mechanism for IDIR users

**Acceptance Criteria**
- [x] navigating to https://dev-aest-sims.apps.silver.devops.gov.bc.ca/aest prompt user to login with IDIR  (replicate /institution and /student)
- [x] Redirect to IDIR IDP (identify provider)
- [x] Redirect back to blank landing page on successful login
- [x] Create keycloak group "aest-user"
- [x] Only allow login if user belongs to group
- [x] Create a new Keycloak client
- [x] Use the Keycloak client name to determine if the user was authenticated using the expected client (on Vue and API).
- [x] Create a decorator at API (similar to the institutions one) to validate if the user is a valid IDIR user that belongs to the expected group
